Formula/openj9.rb

```diff
@@ -0,0 +1,89 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/ibmruntimes/openj9-openjdk-jdk16.git",
+    branch:   "v0.25.0-release",
+    revision: "0c11227a21ace48b7c978e29a347822912a6df9c"
+  version "0.25.0"
+  license any_of: [
+    "EPL-2.0",
+    "Apache-2.0",
+    "GPL-2.0-or-later" => { with: "Classpath-exception-2.0" },
```

Comment:
How to add something like `"LicenseRef-GPL-2.0-or-later" => { with: "OpenJDK-assembly-exception-1.0" },` here?

Reply:
What is `LicenseRef-GPL-2.0-or-later`?

Reply:
Just the same license with another exception.

Reply:
How is it different from
```ruby
"GPL-2.0-or-later" => { with: "OpenJDK-assembly-exception-1.0" },
```
?

Reply:
if use
```
license any_of: [
    "EPL-2.0",
    "Apache-2.0",
    "GPL-2.0-or-later" => { with: "Classpath-exception-2.0" },
    "GPL-2.0-or-later" => { with: "OpenJDK-assembly-exception-1.0" },
  ]
```
`brew audit --strict --new openj9` outputs:
```
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/openj9.rb:10: warning: key "GPL-2.0-or-later" is duplicated and overwritten on line 11
```